param/parameterized.py

```diff
@@ -75,6 +75,22 @@ def logging_level(level):
         param_logger.setLevel(logging_level)
 
 
+@contextmanager
+def batch_watch(parameterized, run=True):
+    """
+    Context manager which batches the watchers on the parameterized.
+    If run=True it will dispatch the accumulated events on the watchers.
```

Comment:
This docstring is a bit obscure; I can't tell what the purpose of this context manager is from it. Can you elaborate?

Reply:
Updated the docstring, hope it's clearer now.